src/carplay.cs

```diff
@@ -573,10 +573,10 @@ interface CPListItem : CPSelectableListItem, NSSecureCoding {
 	interface CPListSection : NSSecureCoding {
 
 		[Export ("initWithItems:header:sectionIndexTitle:")]
-		NativeHandle Constructor (CPListItem [] items, [NullAllowed] string header, [NullAllowed] string sectionIndexTitle);
+		NativeHandle Constructor (ICPListTemplateItem [] items, [NullAllowed] string header, [NullAllowed] string sectionIndexTitle);
```

Comment:
The changes are correct but I believe they are a breaking change which means that we have to provide a path (even when the constructor was wrong). We need to do the following:
```suggestion
#if !XAMCORE_5_0
  		[Export ("initWithItems:header:sectionIndexTitle:")]
  		[Obsolete ("Use  'CPListSection (ICPListTemplateItem [], string, string)' constructor instead.")]
  		NativeHandle Constructor (CPListItem [] items, [NullAllowed] string header, [NullAllowed] string sectionIndexTitle);
#endif
		[Export ("initWithItems:header:sectionIndexTitle:")]
		NativeHandle Constructor (ICPListTemplateItem [] items, [NullAllowed] string header, [NullAllowed] string sectionIndexTitle);
```

Reply:
@dalexsoto  there was a way to allow to methods to have the same selector, right? I don't want to make this change more complicated that it is.